_sass/_components--navbar.scss

```diff
@@ -0,0 +1,68 @@
+$color--navbar-background: $hint-of-red;
+$size--navbar-height: 70px;
+$size--logo-padding-top: 13px;
+$size--nav-links-padding-top: 26px;
+$size--header-title: 20px;
+
+.main-wrapper {
+  padding-top: $size--navbar-height;
+}
+
+.site-navbar {
+  background-color: $color--navbar-background;
+  height: $size--navbar-height;
+  z-index: z('navbar');
+  width: 100%;
+  position: fixed;
+  opacity: 1;
+  transition: all .3s ease;
+
+  // Using headroom.js for scrolling menu effect
+  &.headroom {
+    &--unpinned {
+      z-index: z('navbar-not-active');
+      opacity: 0;
+    }
+    &--top, &--pinned {
+      z-index: z('navbar');
+      opacity: 1;
+    }
+  }
+}
+
+.site-logo {
+  display: inline-block;
+  padding-top: $size--logo-padding-top;
+  padding-left: $space-triple;
+
+  @media #{$just-small-resolutions} {
+    padding-left: 23px;
```

Comment:
this is quite weird, why 23? it looks like a magic number for me. Can you elaborate what you need 23 there?